### PR TITLE
Added version 0.2.0

### PR DESCRIPTION
**Pull request by imjelde**

### CODE REVIEW

File: var/spack/repos/builtin/packages/gitconddb/package.py [hunk 1]
```diff
@@ -18,6 +18,7 @@ class Gitconddb(CMakePackage):
 
     version('master', branch='master')
     version('0.1.1', sha256='024a6867722a3a622ed4327ea7d15641dd48e4e8411bdcc21915e406b3c479a2')
+    version('0.2.0', sha256='dfbaa33581d5c6fac1102668c542b32d14499a30a2793bc55b133aba9f7295fc')
```
Comment (haampie) [inline var/spack/repos/builtin/packages/gitconddb/package.py]: The newest is usually at the top
```suggestion
    version('0.2.0', sha256='dfbaa33581d5c6fac1102668c542b32d14499a30a2793bc55b133aba9f7295fc')
    version('0.1.3', sha256='29ed88498fd7ea3fd24bc80e9bfdec796cbeb584796c94c6c6b0baa368842ad4')
    version('0.1.2', sha256='121ac34d8afffbd97b052cbb10f15e78cef962fe25ded85d88ab26e1677b72b5')
    version('0.1.1', sha256='024a6867722a3a622ed4327ea7d15641dd48e4e8411bdcc21915e406b3c479a2')
```
Maybe also worth adding bug fix releases for 0.1.x

Comment (imjelde) [reply var/spack/repos/builtin/packages/gitconddb/package.py]: Good input! 👍

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/gitconddb/package.py]: > The newest is usually at the top

I thought that was a requirement.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/gitconddb/package.py]: Also confirmed all of the above sha256.

Comment (imjelde) [reply var/spack/repos/builtin/packages/gitconddb/package.py]: My bad. First Spack PR ever. I'll keep it in mind.

Comment (alalazo) [reply var/spack/repos/builtin/packages/gitconddb/package.py]: @imjelde Can you please address this?

**Review by alalazo — APPROVED** [review]

**Pull request merged**